### ISSUE TITLE
Edgar 2013 ancestry

### ISSUE THREAD

**Issue by Lariabeth** [spawaskar-cora/cora-docs#135]
For Edgar 2013 (cranium and mandible) ancestry - it would be nice to have a header for the data entry column that indicates the scores entered should be ASU scores.  Edgar's equations convert these to binary based on the ASU scores - if possible, can we get them to be automatically configured within Cora or would that be an analytics thing ?  I attached screenshots and an Excel sheet demonstrating the scores

I attached screen shots and an Excel sheet of scores. 

![edgar mandible](https://user-images.githubusercontent.com/34746337/38998937-413f0168-43b6-11e8-9443-cc79b923b7c5.JPG)

[Edgar scores 2013.xlsx](https://github.com/spawaskar-cora/cora-docs/files/1928566/Edgar.s
![edgar maxilla](https://user-images.githubusercontent.com/34746337/38998918-3dbe908a-43b6-11e8-83ba-d7e10a94975a.JPG)
cores.2013.xlsx)

**Comment by Lariabeth** [comment]
it is similar - a feature is scored and then converted to a binary based on the expression - we use the ASU method to enter the scores but the ancestry assessment is conducted using equations created by Edgar

**Comment by emilystreetman** [comment]
@SachinPawaskarUNO Yes, it's similar process. We usually need to cite both publications because the "scoring method" is from one publication and the "scores mean this ancestry" is from another. There are a handful of situations like this (e.g., Buikstra and Ubelaker 1994 -> Walker 2008).

@Lariabeth So, would you have people score 0-5 in Turner et al 1990 ASUDAS then 0-1 in Edgar 2013? Turner et al 1990 does not appear to be in CORA at this time. The automated binary coding step may still be a ways off, so it might be good to have a more "manual" two-step data entry plan as is currently in the Hefners. 

I think @clegarde told me that we're holding off somewhat on teeth, not sure what that means for this method enhancement.

**Comment by clegarde** [comment]
This is a bit different than the Hefner 2009 and H&O 2014 scoring. We separate Hefner 2009 and H&O 2014 because we use Hefner 2009 by itself. Although the scores we are doing for Edgar 2013 are following Turner et al 1990, it should remain under the Edgar 2013 method in CoRA. We don't really use Turner et al 1990 by itself.  I think if we separated this process for Edgar 2013, it would cause too much confusion. Like @emilystreetman said, it is common to use someone else's scoring within another method, and we don't separate those out.

If we can calculate the "Binary Score" (I would not call it the Edgar score) in CoRA, that would be great. However, it might be part of the analytics since it is used in more complex formulas?

@emilystreetman , the comment I made about holding off on teeth was just in regards to details for single teeth being entered into CoRA. Does not apply to something like this.

**Comment by Lariabeth** [comment]
@clegarde - I agree with your comments above. I was hoping a separate column could be coded automatically using an "if...then" to convert to binary but wasn't sure how about complexities or whether that would go through analytics - I think it would reduce error if the Turner _et al._ 1990 scoring was entered by the user and the conversion to binary done automatically.  Do we also need the equations for Edgar or is that already entered? Or are we planning on having people do the math themselves?
